backend/cesium_api/views/tiles.py

```diff
@@ -7,11 +7,18 @@
 
 @view_tiles.route("/<int:zoom>/<int:x>/<int:y>.png")
 def get_tile(zoom: int, x: int, y: int):
-    data_dir = current_app.data_dir
-    tile_path = abspath(f"{data_dir}/tiles/{zoom}/{x}/{y}.png")
+    
+    tiles_dir = getattr(current_app, "tiles_dir", "../geo-tiles/tiles")
+    tile_path = abspath(f"{tiles_dir}/tiles/{zoom}/{x}/{y}.png")
+    print(f"[TILE DEBUG] Tile path being served: {tile_path}")
+
+    #tile_path = abspath(f"{data_dir}/tiles/{zoom}/{x}/{y}.png")
+    #print(f"[TILE DEBUG] Tile path being served: {tile_path}")
+
+
 
     try:
-        exists(tile_path)
+        #exists(tile_path)
         return send_file(tile_path)
     except FileNotFoundError:
         return {"err": f"No such tile for {x} by {y} at a zoom of {zoom}"}, 404
```

Comment:
```suggestion
@view_tiles.route("/<int:zoom>/<int:x>/<int:y>.png")
def get_tile(zoom: int, x: int, y: int):
    data_dir = current_app.data_dir
    tile_path = abspath(f"{data_dir}/tiles/{zoom}/{x}/{y}.png")

    try:
        exists(tile_path)
        return send_file(tile_path)
    except FileNotFoundError:
        return {"err": f"No such tile for {x} by {y} at a zoom of {zoom}"}, 404
```

Go ahead and revert this file to what all is on `main`.

I see you added the `tiles_dir` and `DEFAULT_TILES_DIR` fields which is already accomplished via `data_dir` and `DEFAULT_DATA_DIR`.

You may dynamically change what the data-dir is by feeding it in as a cmd line parameter like so:

```bash
python -m cesium_api --data-dir ../geo-tiles/tiles
```